includes/data-stores/class-wcs-subscription-data-store-cpt.php

```diff
@@ -94,10 +94,28 @@ public function __construct() {
 	 * @since 2.2.0
 	 */
 	public function create( &$subscription ) {
+
+		$subscription_status = $subscription->get_status( 'edit' );
+
+		if ( ! $subscription_status ) {
+			$subscription->set_status( 'wc' . apply_filters( 'woocommerce_default_subscription_status', 'pending' ) );
+		}
+
+		/**
+		 * This function is called on the `woocommerce_new_order_data` filter.
+		 * We hook into this function, calling our own filter `woocommerce_new_subscription_data` to allow overriding the default subscription data.
+		 */
+		$new_subscription_data = function ( $args ) {
+			return apply_filters( 'woocommerce_new_subscription_data', $args );
+		};
+		add_filter( 'woocommerce_new_order_data', $new_subscription_data );
```

Comment:
Oh nvm I misread this. This is still necessary because it triggers the `woocommerce_new_subscription_data` hook when creating the subscription via the `parent::create`

Reply:
Is this hook just used for unit testing?